packages/pipeline/test/entities/util.ts

```diff
@@ -15,9 +15,7 @@ const expect = chai.expect;
  * @param entity An instance of a TypeORM entity which will be saved/retrieved from the database.
  */
 export async function testSaveAndFindEntityAsync<T>(repository: Repository<T>, entity: T): Promise<void> {
-    // Note(albrow): We are forced to use an 'as any' hack here because
-    // TypeScript complains about stack depth when checking the types.
-    await repository.save(entity as any);
+    await repository.save<any>(entity);
```

Comment:
This is okay but we should still probably include the comment explaining why it is necessary.